src/engine/GameObjects/MapNode.cxx

```diff
@@ -151,6 +151,13 @@ Layer MapNode::getTopMostActiveLayer() const
   return Layer::NONE;
 }
 
+void MapNode::setNodeTransparency(const float transparencyFactor) const
+{
+  unsigned char alpha = (1 - transparencyFactor) * 255;
```

Comment:
Use 1.0f instead of 1

Consider replacing magic number (255) with constexpr

Reply:
will be done in separate PR, I will add TODO though.